lib/Model/Probes/CircleProbe.php

```diff
@@ -126,6 +130,25 @@ public function includeBackendCircles(bool $include = true): self {
 	}
 
 
+	/**
+	 * @param bool $visit
+	 *
+	 * @return $this
+	 */
+	public function visitSingleCircles(bool $visit = true): self {
```

Comment:
A bit of documentation on what it does? :)

Reply:
So, the provisioning API for Circles (for NC23+) should be able to returns all entities like Circles and Circles generated from Nextcloud Groups. However, with the current status of the code, the Circles App have no way to browse the list of singleIds for Nextcloud Users (as Entities).

This methods visitSingleCircles() allow to add a parameter to the Probe to that the request will also returns singleIds for Nextcloud Users (as a visitor as the current viewer/initiator does not belongs to the Single Circle)

Reply:
No, I meant code documentation :wink: 
Your functions on this are missing docs headers :see_no_evil:

Reply:
i will write this docs in the next big PR about public interface

Reply:
Well, all functions should have a documentation header. It's not about being public or not :wink:

Reply:
@skjnldsv I agree that a short descriptive header is helpful to understand the meaning functions. Though this class is probably a bad example to start. Still, I added documentation headers for the functions: https://github.com/nextcloud/circles/pull/783/commits/369cfe5ed757acf463357b6ed8ad704863eed84c